applications/StructuralMechanicsApplication/structural_mechanics_application_variables.cpp

```diff
@@ -83,7 +83,15 @@ KRATOS_CREATE_VARIABLE(double, SHELL_ELEMENT_BENDING_ENERGY_FRACTION)
 KRATOS_CREATE_VARIABLE(double, SHELL_ELEMENT_SHEAR_ENERGY_FRACTION)
 
 // Membrane variables
-KRATOS_CREATE_VARIABLE(Vector, MEMBRANE_PRESTRESS)
+KRATOS_CREATE_VARIABLE(Matrix, MEMBRANE_PRESTRESS)
+KRATOS_CREATE_VARIABLE(Matrix, PRESTRESS_AXIS_1_GLOBAL)
+KRATOS_CREATE_VARIABLE(Matrix, PRESTRESS_AXIS_2_GLOBAL)
+KRATOS_CREATE_VARIABLE(Matrix, PRESTRESS_AXIS_1_LOCAL)
+KRATOS_CREATE_VARIABLE(Matrix, PRESTRESS_AXIS_2_LOCAL)
+
+// Formfinding
+KRATOS_CREATE_VARIABLE(double, LAMBDA_MAX)
+KRATOS_CREATE_VARIABLE(bool, FORMFINDING)
```

Comment:
are you using this?
If yes, then please rename it to `IS_FORMFINDING`

Reply:
Done. Right now I don't use it but I'm gonna use it in the future.